### ISSUE TITLE
Public cloud: Update an AWS how-to guide based on a blog post

### ISSUE THREAD

**Issue by k-dimple** [canonical/open-documentation-academy#30]
GitHub repo: https://github.com/canonical/ubuntu-cloud-docs

An [AWS how-to guide](https://github.com/canonical/ubuntu-cloud-docs/blob/main/aws/aws-how-to/instances/launch-ubuntu-desktop.rst) within the repo needs to be updated based on this [discourse post](https://discourse.ubuntu.com/t/launch-ubuntu-desktop-mantic-minotaur-on-ec2/40529) which uses a later version of Ubuntu.

**Comment by hippytaffer** [comment]
Hi, I've had a stab at this and created a pull request.

**Comment by k-dimple** [comment]
Thanks @hippytaffer! I'll take a look at it as soon as I get the chance. Meanwhile I have assigned this issue to you, so that others don't think it is still available.

**Comment by geoffreynyaga** [comment]
@k-dimple I have followed the [PR](https://github.com/canonical/ubuntu-cloud-docs/pull/190) 
 and thank you so much for the guidance you gave to @hippytaffer. It genuinely makes me excited to be part of this community. I still see the PR was not merged. Is there anything I can help with this AWS how-to as the current situation stands?

**Comment by k-dimple** [comment]
Hello @geoffreynyaga, thank you so much for the message and the offer to help!

@hippytaffer you are probably busy (like I have been!), would you like Geoffrey to help you finish the PR? 

@geoffreynyaga in the meantime, I would encourage you to look at some of the other open and unassigned issues too. We can always come back to this one later in case we need some help here.

**Comment by hippytaffer** [comment]
Hi both, yes, I have quite a bit on at the moment so that’s fine.

**Comment by k-dimple** [comment]
@hippytaffer thanks for that! @geoffreynyaga please feel free to help complete the PR.

**Comment by k-dimple** [comment]
@geoffreynyaga would you want to take a stab at this one?

**Comment by network-charles** [comment]
Hi @k-dimple, I don't quite understand the issue you want to address.

Are you saying that you don't want a tutorial setup which includes installing TightVNC, but you want one where Ubuntu Mantic is purchased on the marketplace and run as an EC2 instance, as described on the [discourse post](https://discourse.ubuntu.com/t/launch-ubuntu-desktop-mantic-minotaur-on-ec2/40529)?

**Comment by k-dimple** [comment]
Hi @network-charles the existing how-to guide is quite old and mentions the use of 'Ubuntu 16.04 or greater'. However, using TightVNC might not work with the newer versions of Ubuntu, and RDP is the preferred choice. So we want to create a new how-to guide that works on the latest LTS version i.e. on Ubuntu 24.04 (Noble). The process is described in the discourse post (which specifies that it works for both Mantic and Noble) and uses RDP instead of TightVNC.

So, the main idea is to just use the content from the discourse post almost as is and create a how-to guide from it for Noble. The initial steps in the post mention getting an image from the marketplace, but that does not automatically imply that it needs to be purchased. The cost if any will be based on the configuration of the EC2 instance that is used.

**Comment by network-charles** [comment]
Alright.

Since I need to upload screenshots, can I create a folder at “aws/aws-how-to/instances” to upload them?

**Comment by k-dimple** [comment]
Yes, there are few other folders there that store images (one for each how-to-guide that includes images). You can follow the same approach.

Also, I am assuming you know how to clone the project locally to your machine to try things out before submitting a PR :)  The readme has instructions, but if you have any questions feel free to ask here!

**Comment by network-charles** [comment]
Nice, thank you.

Yes, I know how to clone it and open a PR referencing this issue. I won't hesitate to ask here if I feel lost.

**Comment by network-charles** [comment]
Hi @k-dimple, I noticed that the issue is still open. You'll be closing it right?

**Comment by k-dimple** [comment]
Oh I thought I had closed it! Thanks for the ping. Closing now.

**Comment by network-charles** [comment]
Alright
